Frameworks/AEPSwiftUI/Sources/ContentCards/ContentCardUIError.swift

```diff
@@ -0,0 +1,19 @@
+/*
+ Copyright 2024 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import Foundation
+
+/// An enum which describes different errors that ContentCardUI API's can return
+@objc public enum ContentCardUIError: Int, Error {
+    public typealias RawValue = Int
```

Comment:
not sure what this is doing

Reply:
I used them here as an Error when we are not able to read the required proposition for the surface from the Messaging API
https://github.com/adobe/aepsdk-ui-ios/pull/45/files#diff-785bcfab1162e2cd118a33f50be5c6753eb661f1896a4682518c1b968f5ad37dR44